### PR TITLE
Milestone5-Style

### PR DESCRIPTION
**Pull request by janis-jenny**
- Add style to the components: Book, Bookform, Booklist, Progress, Currentchapter

### CODE REVIEW

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @janis-jenny, I am Elias, your reviewer for this PR

Well done so far with your project!

You added all the necessary elements to the design, great job!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] Please align all your elements like in the sample given by zipline, it doesn't have to be perfect but near to the design given to comply with microverse requirements:
image
Right now, they are like this:
image
- [ ] Please enlarge your book title element to make it look like the design given and change the submit button text for "ADD BOOK" to comply with microverse requirements.
- [ ] Your completed percentage moves to the right if the text is large enough, please prevent this from happening:
image
- [ ] Please add more padding/margin in here to make it look like in the original design to comply with microverse requirements:
- Your current design: 
image
- Original design:
image
- [ ] Please diminish the space between this two elements to comply with microverse requirements:
image
- [ ] This one is OPTIONAL but I would suggest you to move your category filter next to the categories text in your navbar to make it look a little better and less confusing.
- [ ] Please add this small separators like in the original design:
image

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap');
+
```
Comment (george-shammar) [inline src/index.css:3]: - [ ] While implementing changes from the last review, you've created significant changes to the style features of the application. Please see below:

Your application now looks like the image below: See the highlighted section. Please, let's fix it.
Screenshot 2021-09-15 at 18-32-27 React App

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @janis-jenny , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------

File: src/index.css (hunk 1 above)
Comment (janis-jenny) [reply src/index.css:3]: In your dev tools is gonna look like that obviously, although this project doesn't require responsive design I tried to fix it with the size of their widths

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @janis-jenny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**